easier68k/core/opcodes/Bcc.py

```diff
@@ -364,4 +364,403 @@ def disassemble_instruction(cls, data: bytes):
+            return True
+
+        return False
+
+    @classmethod
+    def command_matches(cls, command: str):
+        return opcode_util.command_matches(command, 'BGE')
+
+    @classmethod
+    def disassemble_instruction(cls, data: bytes):
+        return bcc_disassemble_instruction(data, BGE, ord('\x0C'))
+
+    @classmethod
+    def from_str(cls, command: str, parameters: str):
+        return bcc_from_str(command, parameters, BGE)
+
+
+class BLT(branch_code):
+    """
+    The BLT opcode compiles down to 6Dxx xxxx xxxx
+    The conditional here is N AND NOT V OR NOT N AND V
+    """
+
+    cond_code = '\x0D'
+
+    def conditional(self, simulator: M68K):
+        n = simulator.get_condition_status_code(ConditionStatusCode.N)
+        v = simulator.get_condition_status_code(ConditionStatusCode.V)
+
+        if n and v:
+            return True
+
+        if not n and not v:
+            return True
+
+        return False
+
+    @classmethod
+    def command_matches(cls, command: str):
+        return opcode_util.command_matches(command, 'BLT')
+
+    @classmethod
+    def disassemble_instruction(cls, data: bytes):
+        return bcc_disassemble_instruction(data, BLT, ord('\x0D'))
+
+    @classmethod
+    def from_str(cls, command: str, parameters: str):
+        return bcc_from_str(command, parameters, BLT)
+
+
+class BGT(branch_code):
+    """
+    The BGT opcode compiles down to 6Exx xxxx xxxx
+    The conditional here is N AND V AND NOT Z OR NOT N AND NOV AND NOT Z
+    """
+
+    cond_code = '\x0E'
+
+    def conditional(self, simulator: M68K):
+        n = simulator.get_condition_status_code(ConditionStatusCode.N)
+        v = simulator.get_condition_status_code(ConditionStatusCode.V)
+        z = simulator.get_condition_status_code(ConditionStatusCode.Z)
+
+        if n and v and not z:
+            return True
+
+        if not n and not v and not z:
+            return True
+
+        return False
+
+    @classmethod
+    def command_matches(cls, command: str):
+        return opcode_util.command_matches(command, 'BGT')
+
+    @classmethod
+    def disassemble_instruction(cls, data: bytes):
+        return bcc_disassemble_instruction(data, BGT, ord('\x0E'))
+
+    @classmethod
+    def from_str(cls, command: str, parameters: str):
+        return bcc_from_str(command, parameters, BGT)
+
+
+class BLE(branch_code):
+    """
+    The BLE opcode compiles down to 6Fxx xxxx xxxx
+    The conditional here is Z OR V AND NOT Z OR NOT N AND NOT V AND NOT Z
+    """
+
+    cond_code = '\x0F'
+
+    def conditional(self, simulator: M68K):
+        n = simulator.get_condition_status_code(ConditionStatusCode.N)
+        v = simulator.get_condition_status_code(ConditionStatusCode.V)
+        z = simulator.get_condition_status_code(ConditionStatusCode.Z)
+
+        if z:
+            return True
+
+        if v and not z:
+            return True
+
+        if not n and not v and not z:
+            return True
+
+        return False
```

Comment:
`return (n != v) and not z`

Reply:
~N & ~V &~Z is not covered by this conditional.

The code I wrote is meant to directly reflect the conditionals in the m68k manual by NXP.
If you would like them more simplified I suppose I could probably bust out a Karnaugh Map for these three variable conditionals. I think this is as concise as It gets though.

Reply:
Do you have the page number of where this is in the manual? I'm only able to find the checks on floating point numbers.

Two other sources I like to use are http://68k.hax.com/Bcc and http://wpage.unina.it/rcanonic/didattica/ce1/docs/68000.pdf because I find them much more readable. Both of these list `Z + N*!V + !N*V`

We can simplify the expression `N*!V + !N*V` to be `N != V`:
```
+---+---+-------------+
| N | V | N*!V + !N*V |
+---+---+-------------+
| 0 | 0 | 0           |
| 0 | 1 | 1           |
| 1 | 0 | 1           |
| 1 | 1 | 0           |
+---+---+-------------+
```

Checking for `not Z` after line 743 is redundant. We could actually reduce the whole thing to `return (n != v) or z`

```
+---+---+---+-----------------+
| N | V | Z | Z + N*!V + !N*V |
+---+---+---+-----------------+
| 0 | 0 | 0 | 0               |
| 0 | 0 | 1 | 1               |
| 0 | 1 | 0 | 1               |
| 0 | 1 | 1 | 1               |
| 1 | 0 | 0 | 1               |
| 1 | 0 | 1 | 1               |
| 1 | 1 | 0 | 0               |
| 1 | 1 | 1 | 1               |
+---+---+---+-----------------+
```